content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/_cq_dialog/.content.xml

```diff
@@ -95,6 +95,18 @@
                                                         value="h6"/>
                                                 </items>
                                             </defaulttypes>
+                                            <linkURL
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/pathfield"
+                                                fieldDescription="Make the title a link to another resource."
```

Comment:
For the sling:resourceType we should better use: cq/gui/components/coral/common/form/pagefield instead of granite/ui/components/coral/foundation/form/pathfield

Reply:
Why not use a `/granite/ui/components/coral/foundation/form/pathbrowser`?

Reply:
@kwin - the `granite/ui/components/coral/foundation/form/pathbrowser` is superseded by `granite/ui/components/coral/foundation/form/pathfield`, but are conceptually the same.

We think it may be more useful to limit the picker to pages in this case rather than any path. It's still possible to paste any path (content path, external URLs, anchors etc.) in the text field.

Do you see a use case where general path picking would be more useful?

Reply:
No, please ignore my previous comment. I just figured out that both are now actually the same (from a feature perspective). My knowledge from the restriction of the `pathfield` not having a browser button is probably from an old AEM version.